features/sign-up.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Sign-up
+    As a User
+    I want to access the platform with my credentials
+    So that i can see my data streams
+
+    Scenario: Successful User Sign-up
+        Given I am at the sign-up page
+        When I fill the field "E-mail" with "sauron@mordor.arda"
```

Comment:
this is temporary, right?

Reply:
No. Why should it be?

Reply:
I was thinking wrongly.

However I think we could abstract the variable-like strings. Example:

```
Scenario Outline: controlling order
  Given there are <start> cucumbers
  When I eat <eat> cucumbers
  Then I should have <left> cucumbers

  Examples:
    | start | eat | left |
    |  12   |  5  |  7   |
```

Reply:
I don't see much improvement from what we have